### PR TITLE
1108 retrieve env vars

### PR DESCRIPTION
**Pull request by ricardovdheijden**
#1108 

- Applies name changes in environment variables
- Using `getEnvironmentVariables` helper to retrieve environment variables

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Updating library version in package-lock.json.
+
+Since the version of dependency `@orchestrator-ui/orchestrator-ui-components` is `*` for proper usage in our Turborepo, we still need to update package-lock.json to make the build command build the app with the latest library version.
```
Comment (DutchBen) [inline README.md:3]: For a user that's unfamiliar with the library repo this is confusing I think. Also it only applies if you have already have  a previous installation. I would remove this or expand it to include those conditions

File: breaking-changes.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+| ---------------------------- | ----------------------------- |
+| AUTH_ACTIVE                  | OAUTH2_ACTIVE                 |
+| NEXTAUTH_CLIENT_ID           | OAUTH2_CLIENT_ID              |
+| NEXTAUTH_CLIENT_SECRET       | OAUTH2_CLIENT_SECRET          |
+| NEXTAUTH_ID                  | NEXTAUTH_PROVIDER_ID          |
+| NEXTAUTH_ID                  | NEXTAUTH_PROVIDER_NAME        |
+| NEXTAUTH_ISSUER              | OIDC_CONF_FULL_WELL_KNOWN_URL |
+| NEXTAUTH_WELL_KNOWN_OVERRIDE | OIDC_CONF_FULL_WELL_KNOWN_URL |
+
+Note: `NEXTAUTH_ID` is split up in 2 new environment variables. The `NEXTAUTH_ISSUER` and `NEXTAUTH_WELL_KNOWN_OVERRIDE` are replaces by one new environment variable.
```
Comment (DutchBen) [inline breaking-changes.md:17]:
```suggestion
Note: `NEXTAUTH_ID` is split up in 2 new environment variables. The `NEXTAUTH_ISSUER` and `NEXTAUTH_WELL_KNOWN_OVERRIDE` are replaced by one new environment variable.
```

**Review by DutchBen — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ricardovdheijden) [reply README.md:3]: Renaming the file to have it not shown direct;y when visiting the repository's webpage

File: configuration/configuration.ts [hunk 3]
```diff
@@ -1,51 +1,49 @@
...
-export const DEFAULT_WORKFLOW_INFORMATION_LINK_URL = 'http://localhost:8080';
-
 export const getInitialOrchestratorConfig = (): OrchestratorConfig => {
-    const orchestratorGraphqlBaseUrl =
-        process.env.ORCHESTRATOR_GRAPHQL_HOST &&
-        process.env.ORCHESTRATOR_GRAPHQL_PATH
-            ? `${process.env.ORCHESTRATOR_GRAPHQL_HOST}${process.env.ORCHESTRATOR_GRAPHQL_PATH}`
-            : DEFAULT_GRAPHQL_CORE_ENDPOINT;
+    const {
+        USE_THEME_TOGGLE,
```
Comment (ricardovdheijden) [inline configuration/configuration.ts:9]: Todo: update .env.example

**Review by Georgi2704 — APPROVED** [review]

**Pull request merged**